_posts/convars/2022-02-02-mom_hud_status_enable.md

```diff
@@ -0,0 +1,13 @@
+---
+title: mom_hud_status_enable
+category: var
+tags:
+  - hud
+  - map
+  - map info
```

Comment:
Missed this in the initial review but this hud element isnt tied to mapinfo, so can omit these 2 tags.